src/test/java/teammates/test/cases/testdriver/HtmlHelperTest.java

```diff
@@ -135,4 +135,20 @@ public void testConvertToStandardHtmlPart() throws Exception {
         
         //other cases are tested in testComparison
     }
+    
+    @Test
+    public void testReplacement() throws IOException {
```

Comment:
I dont understand. Are we testing the lower level functions that consitute Godmode over here? If yes, I think this test should be a little more stringent. Eg: Check the expected string after line 142, 143 and 146. The ```assertSameHtml``` statement in line 147 can be removed